client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -27,11 +27,86 @@
         style="max-height: 52vh;"
       />
     </b-form-group>
+
+    <div>
+      <b-form-group
+        :label="$t('sanitizers.label')"
+        label-size="lg"
+      >
+        <template #label>
+          <div
+            class="d-flex"
+          >
+            Condition Fields
+
+            <b-button
+              variant="link"
+              class="p-0 ml-1 mr-auto"
```

Comment:
I would prefer if you did ml auto on the examples section

Reply:
![image](https://user-images.githubusercontent.com/31524469/214264985-ed23f4ef-6c1d-4599-8c98-9b7b4c6aede5.png)

This is what ml-auto looks like.